src/content/index.tsx

```diff
@@ -24,6 +31,62 @@ export const highAffinityContentArray: ContentPanelProps[] = [
             </>
         ),
         callToAction: "What happens to the molecules once you press play?",
+        section: Section.Introduction,
+        layout: LayoutType.LiveSimulation,
+    },
+    {
+        content: (
+            <>
+                As the simulation plays, the molecules move by a random walk.
+                Watch what happens when they bump into each other. What do you
+                think the "Reaction events over time" graph is showing?
+            </>
+        ),
+        callToAction:
+            "After you've observed the simulation, click 'Lab view' to see what the cuvette looks like now.",
+        section: Section.Introduction,
+        layout: LayoutType.LiveSimulation,
+    },
+    {
+        content: (
+            <>
+                The clear liquid is slowly turning yellow as the simulation
+                progresses. Can you estimate the concentration of <AB /> after
+                about 20 simulated {MICRO}seconds have elapsed?
+            </>
+        ),
+        callToAction: "Click 'Molecular view' to switch back.",
+        section: Section.Introduction,
+        layout: LayoutType.LiveSimulation,
+    },
+    {
+        title: "Start the Experiment",
+        content: (
+            <>
+                Now we're going to use this simulation to make measurements.
+                We're going to increase the timestep so the experiments are
+                fast.
+            </>
+        ),
+        actionButton: <StartExperiment />,
+        callToAction:
+            "Click the 'Start experiment' button to reset the simulation and begin by pressing play!",
+        section: Section.Introduction,
+        layout: LayoutType.LiveSimulation,
+    },
+    {
+        content: (
+            <>
+                Now we're going to use this simulation to make measurements.
```

Comment:
Nit: repeat use of "we're going to" feels awkward...

```suggestion
                Now, let's use this simulation to make measurements.
```